### ISSUE TITLE
Cloud build error: no Go files in /layers/google.go.appengine_gomod/srv

### ISSUE THREAD

**Issue by DaanvandeHaar** [google-github-actions/deploy-appengine#17]
### TL;DR
Google cloud build fails with error message: no Go files in /layers/google.go.appengine_gomod/srv

**Expected behavior**
A successful  cloud build

**Observed behavior**
this error message: no Go files in /layers/google.go.appengine_gomod/srv

### Reproduction

**Action YAML**

name: Go

on:
  release:
    types: [created]

jobs:

  build:
    name: Deploy
    runs-on: ubuntu-latest
    steps:

    - name: Set up Go 1.x
      uses: actions/setup-go@v2
      with:
        go-version: ^1.13

    - name: Check out code into the Go module directory
      uses: actions/checkout@v2

    - name: Get dependencies
      run: |
        go get -v -t -d ./...
        if [ -f Gopkg.toml ]; then
            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
            dep ensure
        fi
    - name: Build
      run: go build -v ./...

    - name: Test
      run: go test -v ./...

    - uses: google-github-actions/setup-gcloud@master
      with:
        version: '290.0.1'
        service_account_key: ${{ secrets.GCP_SA_KEY }}
        export_default_credentials: true
    - id: Deploy
      uses: google-github-actions/deploy-appengine@main
      with:
        project_id: ${{ secrets.project_id }}

**Repository**
https://github.com/DaanvandeHaar/BEP_Lingo
